src/language/dictionary/JMDict/Spelling.java

```diff
@@ -14,13 +13,15 @@
     private boolean isKanji;
     private String spelling;
     private Spelling[] dependencies;
+    private List<Spelling> readings;
```

Comment:
I suppose... Though that's still a lot of null pointers (384 842 * 4 bytes per pointer = ~1.5MB of nulls).

I suppose I'll merge it since it doesn't break anything and mark it for improvement later.